dune-project

```diff
@@ -15,6 +15,6 @@
  (name api-watch)
  (synopsis "Libraries and tools to keep watch on your OCaml lib's API changes")
  (depends
-  (ocaml (and (>= 4.14.0) (< 5.0.0)))
+  (ocaml (and (>= 4.14.0) (< 5.2.0)))
```

Comment:
This is still incorrect, you also need to raise the lower bound from `4.14.0` to `5.1.0`. Here you are allowing all of `4.14`, `5.0` and `5.1`.

Reply:
Sorry to miread your instruction